vendor/modules.txt

```diff
@@ -195,7 +195,7 @@ github.com/google/uuid
 # github.com/hashicorp/errwrap v1.1.0
 ## explicit
 github.com/hashicorp/errwrap
-# github.com/hashicorp/go-azure-helpers v0.30.0
+# github.com/hashicorp/go-azure-helpers v0.31.0
```

Comment:
Just a reminder that we'll need to include https://github.com/hashicorp/go-azure-helpers/pull/115 once it's merged/released.